.eslintrc

```diff
@@ -22,7 +22,7 @@ env:
   es6: true
   worker: false
   amd: true
-  mocha: false
+  mocha: true
```

Comment:
So as to not pollute the global linting configuration in service of tests, we have an `.eslintrc` file in the test directory [here](https://github.com/cfpb/consumerfinance.gov/blob/main/test/.eslintrc). The changes in this file should be reverted and `mocha: true` and `Cypress: true` should be added to test eslint config.

Reply:
Thanks for letting me know. I reverted the change with `mocha`.

Reply:
Thanks! Looks like `Cypress: true` still needs to make it over too.

Reply:
And also `cy: true`. This happened in an earlier PR, but mind moving it too? [This change](https://github.com/cfpb/consumerfinance.gov/commit/c60e6c51a8070f938e4c88bf517ae9354d1fedca#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822) should be reverted in `./.eslintrc` and moved to `./test/.eslintrc`

Reply:
I completed this task and moved Cypress globals to test lint config.